### PR TITLE
Update contributors and metadata title; regenerate CLDF files

### PR DESCRIPTION
**Pull request by tresoldi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,4 +31,16 @@ Conceptlists in Concepticon:
...
-- **Inventory size (avg):** 49.41
\ No newline at end of file
+- **Inventory size (avg):** 49.41
+
+# Contributors
+
+Name | GitHub user | Description | Role
+--- | --- | --- | --- |
+Tiago Tresoldi | @tresoldi | patron, orthography | Autor
+Johann-Mattis List | @lingulist | code, orthography, concepts | Autor
```
Comment (xrotwang) [inline README.md:41]: This should be `Author` - with an `h`.

Comment (tresoldi) [reply README.md:41]: Yes, sorry for these...

**Comment by LinguList** [comment]
Is there a test for the title now?

**Comment by xrotwang** [comment]
@LinguList in pylexibank, you mean?

**Comment by LinguList** [comment]
@xrotwang, just in `test.py`, my new version is like this:

```python
def test_valid_title(cldf_dataset, cldf_logger):
    title = cldf_dataset.metadata_dict['dc:title']
    assert title.startswith('CLDF dataset derived from')
    assert title[-4:].isdigit()
    author = title.split('CLDF dataset derived from')[1].split('"')[0]
    _, short_title, year_ = title.split('"')
    year = year_[6:]
    assert year.isdigit()
    assert author.endswith("'s ")
```

Or can this also be added to pylexibank?

**Comment by xrotwang** [comment]
Ok, I see. I guess I would add the main logic of the test to `pylexibank.util` or similar, and call it from the dataset specific tests. Calling it generally from `cldfbench lexibank.check` would probably lead to too many false positives.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Contributors
+
+Name | GitHub user | Description | Role
+--- | --- | --- | --- |
+Tiago Tresoldi | @tresoldi | patron, orthography | Author
+Johann-Mattis List | @lingulist | code, orthography, concepts | Author
+Robert Forkel | @xrotwang | code, integration | Author
+Christoph Rzymski | @chrzyki | code, integraration | Author
+Timotheus Adrianus Bodt | | | DataCurator, Distributor
```
Comment (LinguList) [inline CONTRIBUTORS.md:9]: Lieberherr is also missing here, I would say.

File: cldf/parameters.csv [hunk 3]
```diff
@@ -48,7 +48,7 @@ ID,Name,Concepticon_ID,Concepticon_Gloss
 47_house,house,1252,HOUSE
 48_in,in,1460,IN
 49_knee,knee,1371,KNEE
-50_toknow,to know,1410,KNOW (SOMETHING)
+50_toknow,to know,3626,KNOW
```
Comment (LinguList) [inline cldf/parameters.csv:51]: Who changed this concepticon mapping here? We could have just asked Tim, so we don't need to go for a less specific concept. This is just emphasizing that we need to stop post-editing the concept lists in huge bulk edits, as so many details, where we know the people and can ask them, just get lost here.

Comment (LinguList) [reply cldf/parameters.csv:51]: Just raised an issue, ignore for this PR: https://github.com/concepticon/concepticon-data/issues/820

File: metadata.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-    "title": "Sub-grouping Kho-Bwa",
+    "title": "CLDF dataset derived from Lierberherr and Bodt's \"Comparative Wordlists of Kho-Bwa\" from 2017",
```
Comment (LinguList) [inline metadata.json:2]: Really, `Lierberherr`? Can't we pay a bit more attention to the spelling, please?

**Comment by LinguList** [comment]
@xrotwang, okay, one small edit needed to the assert:
```python
    assert author.endswith("'s ") or author.endswith("s' ")
```

Should I add the test later, or should I leave this to you?

**Comment by LinguList** [comment]
Yes, in pylexibank.

**Comment by xrotwang** [comment]
@LinguList I'll add it to `pylexibank`. But the individual `test.py`s can only call it then after the next pylexibank release.

File: metadata.json (hunk 4 above)
Comment (tresoldi) [reply metadata.json:2]: Sorry

**Comment by tresoldi** [comment]
@xrotwang I have been running those in sync with the `pylexibank master`, should I keep doing it?

**Comment by xrotwang** [comment]
@tresoldi yes, you can keep doing that. buildbot will need a release of pylexibank, though.

**Comment by LinguList** [comment]
Okay, I think it is fine: if we find inconsistencies later, we can still edit them another time, but it will be nice to have some regular tests here.

**Pull request merged**